docs/tutorials/index.rst

```diff
@@ -21,88 +21,125 @@ handbook`_ and the `Astropy Hands-On Tutorial`_.
 
 .. _tutorials_notebooks:
 
-Notebooks
----------
+Getting started
+---------------
 
 .. toctree::
    :hidden:
 
    ../notebooks/analysis_1.ipynb
    ../notebooks/analysis_2.ipynb
+
+The following tutorials show how to use gammapy to perform a complete data analysis,
+here a simple 3D cube analysis of the Crab. They show the gammapy workflow from data selection
+to data reduction and finally modeling and fitting.
+
+First, we show how to do it with the high level interface in configuration-driven approach.
+The second tutorial exposes the same analysis, this time using the medium level API, showing
+what is happening 'under-the-hood':
+
+- `Configuration driven analysis <../notebooks/analysis_1.html>`__ | *analysis_1.ipynb*
+- `Lower level analysis <../notebooks/analysis_2.html>`__ | *analysis_2.ipynb*
+
+
+Core tutorials
+--------------
+
+.. toctree::
+   :hidden:
+
    ../notebooks/cta.ipynb
    ../notebooks/hess.ipynb
    ../notebooks/fermi_lat.ipynb
    ../notebooks/cta_data_analysis.ipynb
    ../notebooks/analysis_3d.ipynb
-   ../notebooks/analysis_mwl.ipynb
    ../notebooks/simulate_3d.ipynb
-   ../notebooks/detect.ipynb
    ../notebooks/spectrum_analysis.ipynb
    ../notebooks/sed_fitting.ipynb
    ../notebooks/light_curve.ipynb
    ../notebooks/light_curve_flare.ipynb
-   ../notebooks/cta_sensitivity.ipynb
    ../notebooks/spectrum_simulation.ipynb
    ../notebooks/modeling_2D.ipynb
    ../notebooks/ring_background.ipynb
-   ../notebooks/overview.ipynb
-   ../notebooks/maps.ipynb
-   ../notebooks/modeling.ipynb
-   ../notebooks/models.ipynb
-   ../notebooks/catalog.ipynb
 
-**Getting started**
+The following tutorials expose common tasks.
 
-The following tutorials show the same 3D cube analysis of the Crab with the high level
-interface and with the lower level API:
-
-- `First analysis <../notebooks/analysis_1.html>`__ | *analysis_1.ipynb*
-- `Second analysis <../notebooks/analysis_2.html>`__ | *analysis_2.ipynb*
-
-**What data can I analyse?**
+*Accessing and exploring DL3 data*
 
+- `Overview <../notebooks/overview.html>`__  | *overview.ipynb*
 - `CTA with Gammapy <../notebooks/cta.html>`__ | *cta.ipynb*
 - `H.E.S.S. with Gammapy <../notebooks/hess.html>`__ |  *hess.ipynb*
-- `Fermi-LAT with Gammapy <../notebooks/fermi_lat.html>`__ | *fermi_lat.ipynb*
 
-**Analyses**
-
-*3-dim sky cube analysis*
+*1D spectral analysis*
```

Comment:
Nitpicking, but 1-dim like in the other two

Reply:
done